File: Java/Shirt.java
class Shirt extends Clothing
{
    //atribut
    private String color;
    private String types;

    //constructor
    public Shirt()
    {
        this.color = "";
        this.types = "";
    }
    public Shirt(String color, String types)
    {
        this.color = color;
        this.types = types;
    }

    //setter
    public void setColor(String color)
    {
        this.color = color;
    }
    public void setTypes (String types)
    {
        this.types = types;
    }

    //getter
    public String getColor()
    {
        return this.color;
    }
    public String getTypes()
    {
        return this.types;
    }

}

File: PHP/Shirt.php
<?php
/* 
    Saya Sabila Rosad NIM 2106000 mengerjakan latihan 2
    dalam mata kuliah Desain Pemrograman Berorientasi Objek
    untuk keberkahanNya maka saya tidak melakukan
    kecurangan seperti yang telah dispesifikasikan 
*/
//library
require('Clothing.php');

//deklarasi class Shirt yang menjadi anak class Clothing
class Shirt extends Clothing
{
     //private atribut
     private $color;
     private $types;
     
     
     //constructor
     public function __construct($color = '', $types = '')
     {
         $this->color = $color;
         $this->types = $types;
     }
 
     //setter
     public function setColor($color)
     {
         $this->color = $color;
     }
     public function setTypes($types)
     {
         $this->types = $types;
     }
     
     
     //getter
     public function getColor()
     {
         return $this->color;
     }
     public function getTypes()
     {
         return $this->types;
     }
     
     
}
?>